.asf.yaml

```diff
@@ -57,5 +57,5 @@ notifications:
   issues: issues@hugegraph.apache.org
   issues_status: dev@hugegraph.apache.org
   issues_comment: issues@hugegraph.apache.org
-  discussions_status: dev@hugegraph.apache.org
+  discussions: dev@hugegraph.apache.org
   discussions_comment: issues@hugegraph.apache.org
```

Comment:
@Z-HUANT we need also remove `discussions_comment` for same problem

Reply:
> @Z-HUANT`discussions_comment`对于同样的问题，我们还需要删除

Pull request has been submitted. https://github.com/apache/incubator-hugegraph/pull/2250